tap_playvox/endpoints.py

```diff
@@ -22,6 +22,20 @@
             'id': 'id',
             'date': 'date'
         }   
+    },
+    'schedule_metrics': {
+        'persist': True,
+        'path': 'api/schedule/metrics',
+        'data_key': 'data',
+        'metric_key': 'dates',
+        'user_key':  'users',
+        'channel_key': 'channels', 
+        'pk': ['id', 'date'],
```

Comment:
why is the pk id and date? shouldn't it just be id?
if there's an existing record in the table: update it by id
if there isn't: create by id

otherwise there'll be dupe schedule metrics by id.

Reply:
There is no unique scheudle_metric ID. The data from this endpoint contains Agent ID ('id') and date, which will need to be used in combination to identify dups. In this case, for every agent for a given day, they can only have one set of metrics.